cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ProgramLifecycleService.java

```diff
@@ -391,6 +394,33 @@ public RunId run(ProgramId programId, Map<String, String> overrides, boolean deb
     return runInternal(programId, userArgs, sysArgs, debug);
   }
 
+  /**
+   * Starts programs which were suspended between startTime and endTime.
+   *
+   * @param applicationId the application to restart programs in
+   * @param startTimeSeconds earliest stop time in seconds of programs to restart
+   * @param endTimeSeconds latest stop time in seconds of programs to restart
+   * @return {@link Set<RunId>}
```

Comment:
Should have a doc for what's get returned.

Reply:
Done, thanks for the review!